pages/fangninghe.md

```diff
@@ -5,4 +5,4 @@
 >I am currently a graduate student of Software Engineering major at Northeastern University. I like to take responsibilities and take teamwork seriously. I am looking forward to working with you and learning from you guys!
 
 * Email: fangning.he.alex@gmail.com
-* LinkedIn: www.linkedin.com/in/fangning-he-62180b106
+* Personal Page: https://fangninghe.github.io/#!pages/fangninghe.md
```

Comment:
ooops
Is this a regression?

I was expecting like
https://www.linkedin.com/in/fangning-he-62180b106
or
[www.linkedin.com/in/fangning-he-62180b106](https://www.linkedin.com/in/fangning-he-62180b106)
which in Markdown is written
`[www.linkedin.com/in/fangning-he-62180b106](https://www.linkedin.com/in/fangning-he-62180b106)`
but not a total URL change ;)